hoomd/BondedGroupData.h

```diff
@@ -108,6 +108,17 @@ void serialize(Archive& ar, hoomd::group_storage<4>& s, const unsigned int versi
     ar& s.tag[2];
     ar& s.tag[3];
     }
+//! Serialization of group_storage<6> (meshtriangles)
+template<class Archive>
+void serialize(Archive& ar, hoomd::group_storage<6>& s, const unsigned int version)
+    {
+    ar& s.tag[0];
+    ar& s.tag[1];
+    ar& s.tag[2];
+    ar& s.tag[3];
+    ar& s.tag[4];
+    ar& s.tag[5];
```

Comment:
```suggestion
    ar(s.tag[0], s.tag[1], s.tag[2], s.tag[3], s.tag[4], s.tag[5]);
```
Preferred way to do this in cereal (https://uscilab.github.io/cereal/assets/doxygen/classcereal_1_1InputArchive.html#a88c7141a24a934978b1ef9894215e67f)